src/Discord/Endpoint.php

```diff
@@ -107,6 +99,13 @@ class Endpoint
     // DELETE
     public const USER_MESSAGE_REACTION = self::CHANNEL.'/messages/:message_id/reactions/:emoji/:user_id';
 
+    // GET
+    public const MESSAGE_POLL = self::CHANNEL.'/polls/:message_id';
```

Comment:
```suggestion
    protected const MESSAGE_POLL = self::CHANNEL.'/polls/:message_id';
```

Don't make it public if the endpoint is not allowed to be hit by the bot, it is counted as undocumented endpoint